3_Implementation/documentation/html/search/enumvalues_0.js

```diff
@@ -0,0 +1,36 @@
+  ['unity_5fdisplay_5fstyle_5fint8_921',['UNITY_DISPLAY_STYLE_INT8',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a78060e131071395f2d2aac77f261ed5f',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fuint_922',['UNITY_DISPLAY_STYLE_UINT',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a7901b0c93cfa82ce8c7203132d4183ae',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fuint16_923',['UNITY_DISPLAY_STYLE_UINT16',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4ab2772ac78a9385b34827680629f30a57',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fuint32_924',['UNITY_DISPLAY_STYLE_UINT32',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a6658cd9d7d136ee81a34600b7385567e',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fuint8_925',['UNITY_DISPLAY_STYLE_UINT8',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a6ea09dbdaa4fe235b7422c850a4b14f8',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5funknown_926',['UNITY_DISPLAY_STYLE_UNKNOWN',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a8623f549d1b8e9da499f091eae8f86da',1,'unity_internals.h']]],
+  ['unity_5fequal_5fto_927',['UNITY_EQUAL_TO',['../unity__internals_8h.html#af032691a9e32795aa06770185ac14a48ae634f01c386e7cf5adb87fdff10d0ef7',1,'unity_internals.h']]],
+  ['unity_5ffloat_5finvalid_5ftrait_928',['UNITY_FLOAT_INVALID_TRAIT',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a6e57f534ca4e720c16a30a19dbf530b2',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fdet_929',['UNITY_FLOAT_IS_DET',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a9df079c54906458c9d1d99d2ac26275a',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5finf_930',['UNITY_FLOAT_IS_INF',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a9ceb93a4a5ce94726a153ca20877c447',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fnan_931',['UNITY_FLOAT_IS_NAN',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a43b25527bc46de77748a152b4be49057',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fneg_5finf_932',['UNITY_FLOAT_IS_NEG_INF',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a4ae18ad2eb5577fbbb61396e7de329cc',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fnot_5fdet_933',['UNITY_FLOAT_IS_NOT_DET',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a87c2784971818301fc117b2ec02e339e',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fnot_5finf_934',['UNITY_FLOAT_IS_NOT_INF',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a55d10c41853433a52e4bcf114370cc3c',1,'unity_internals.h']]],
+  ['unity_5ffloat_5fis_5fnot_5fnan_935',['UNITY_FLOAT_IS_NOT_NAN',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a6cd53722882f0d01dea4da8d2c148898',1,'unity_internals.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unity_5ffloat_5fis_5fnot_5fnan_935",['UNITY_FLOAT_IS_NOT_NAN',['../unity__internals_8h.html#a82d42288d4c610f7e1db67f10c0bb572a6cd53722882f0d01dea4da8d2c148898',1,'unity_internals.h']]],
```